clouddq/classes/bigquery_client.py

```diff
@@ -0,0 +1,177 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+import logging
+import re
+
+from pathlib import Path
+from string import Template
+
+import google.auth
+import google.auth.transport.requests
+
+from google.api_core.exceptions import Forbidden
+from google.api_core.exceptions import NotFound
+from google.auth import impersonated_credentials
+from google.auth.credentials import Credentials
+from google.cloud import bigquery
+from google.oauth2 import id_token
+from google.oauth2 import service_account
+
+
+logger = logging.getLogger(__name__)
+
+TARGET_SCOPES = [
+    "https://www.googleapis.com/auth/cloud-platform",
+    "https://www.googleapis.com/auth/userinfo.email",
```

Comment:
is the user info scope required?

Reply:
you are right this is not needed. I will remove it